[2024-01-27T12:37:30.209+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sales_forecast_pipeline.load_to_database manual__2024-01-27T12:34:41.251258+00:00 [queued]>
[2024-01-27T12:37:30.224+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sales_forecast_pipeline.load_to_database manual__2024-01-27T12:34:41.251258+00:00 [queued]>
[2024-01-27T12:37:30.225+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-01-27T12:37:30.263+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): load_to_database> on 2024-01-27 12:34:41.251258+00:00
[2024-01-27T12:37:30.281+0000] {standard_task_runner.py:60} INFO - Started process 30407 to run task
[2024-01-27T12:37:30.288+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'sales_forecast_pipeline', 'load_to_database', 'manual__2024-01-27T12:34:41.251258+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/sales_forecast_pipeline.py', '--cfg-path', '/tmp/tmpimxo8cxj']
[2024-01-27T12:37:30.291+0000] {standard_task_runner.py:88} INFO - Job 36: Subtask load_to_database
[2024-01-27T12:37:30.357+0000] {task_command.py:423} INFO - Running <TaskInstance: sales_forecast_pipeline.load_to_database manual__2024-01-27T12:34:41.251258+00:00 [running]> on host fb667f005a5b
[2024-01-27T12:37:30.479+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sales_forecast_pipeline' AIRFLOW_CTX_TASK_ID='load_to_database' AIRFLOW_CTX_EXECUTION_DATE='2024-01-27T12:34:41.251258+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-27T12:34:41.251258+00:00'
[2024-01-27T12:37:32.574+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-27T12:37:32.602+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=sales_forecast_pipeline, task_id=load_to_database, execution_date=20240127T123441, start_date=20240127T123730, end_date=20240127T123732
[2024-01-27T12:37:32.718+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-27T12:37:32.768+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
